pkgs/build-support/setup-hooks/breakpoint-hook.sh

```diff
@@ -0,0 +1,9 @@
+breakpointHook() {
+    local red='\033[0;31m'
+    local no_color='\033[0m'
+
+    echo -e "${red}build failed in ${curPhase} with exit code ${exitCode}${no_color}"
+    printf "To attach install cntr into your profile and run the following command as root:\n\n"
+    sh -c "echo '   cntr attach -t command cntr-${out}'; while true; do sleep 99999999; done"
+}
+failureHooks+=(breakpointHook)
```

Comment:
Maybe this should also add an exitHook that fails the build to avoid accidental inclusion in a pr?

Reply:
I would rather not do that because:

```nix
propagatedBuildInputs = [ breakpointHook ];
```

can be quite useful as well.
If this really becomes a problem we can still grep with ofBorg in the git diff.

Reply:
right, nevermind then